Exec/science/wdmerger/initial_model.H

```diff
@@ -87,11 +71,11 @@ void interpolate_3d_from_1d (initial_model::model& model, Real radius,
 
                     Real dist = std::sqrt(x * x + y * y + z * z);
 
-                    state.rho = state.rho + interpolate::interpolate(dist, problem::initial_model_npts, model.r, model.rho);
-                    state.T   = state.T   + interpolate::interpolate(dist, problem::initial_model_npts, model.r, model.T);
+                    state.rho = state.rho + interpolate::interpolate(dist, problem::initial_model_npts, &model.r(0), &model.state(0, model::idens));
```

Comment:
why don't you use the interpolate function built into the model parser?

Reply:
That's the next PR. Didn't want to change too much in one PR.